src/dolphin/workflows/displacement.py

```diff
@@ -234,8 +234,7 @@ def run(
     # ##############################################
 
     ts_opts = cfg.timeseries_options
-    # Skip if we only have 1 unwrapped, or if we didn't ask for inversion/velocity
-    if len(unwrapped_paths) > 1 and (ts_opts.run_inversion or ts_opts.run_velocity):
+    if not ts_opts.run_inversion:
```

Comment:
Could you switch back these changes? I think this happened from me merging the other fixes in the last PR, which you may not have had in your branch

Reply:
Ok yeah I see this has already been addressed. I've adjusted this commit accordingly.